backend/transaction-service/src/transactions/migrations.ts

```diff
@@ -9,14 +9,13 @@ export async function up(database: Database): Promise<void> {
     await database.exec(`
```

Comment:
This new library properly checks for syntax errors regardless of the method used. The screenshot in the PR description is from this `exec` method. I chose to maintain `exec` here since the examples used by these libraries indicate that migrations are typically run using `exec`. The main difference between `exec` and `prepare` detailed here:
https://github.com/WiseLibs/better-sqlite3/blob/master/docs/api.md#execstring---this

I'm really not sure why the base `sqlite` and `sqlite3` libs weren't performing query validation. No matter what junk I put in the query, it would output that to the `tmp` db file